src/main/steps/edgeCase/edgeCaseSequence.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/order */
+import { Sections, Step } from '../constants';
+import { APPLICATION_CONFIRMATION, CHECK_YOUR_ANSWERS, CITIZEN_HOME_URL, DATA_VERIFICATION, START_HOME } from '../urls';
+
+export const EdgeCaseSequence: Step[] = [
+  {
+    url: START_HOME,
+    showInSection: Sections.EdgeCase,
+    getNextStep: () => CITIZEN_HOME_URL,
```

Comment:
is it possible to update this CITIZEN_HOME_URL to CASE_SEARCH_URL

Reply:
done